MapboxNavigation/LanesView.swift

```diff
@@ -67,30 +67,27 @@ open class LanesView: UIView {
     }
     
     func update(for currentLegProgress: RouteLegProgress) {
-        guard let step = currentLegProgress.upComingStep else { return }
-        guard !currentLegProgress.userHasArrivedAtWaypoint else { return }
         let durationRemaining = currentLegProgress.currentStepProgress.durationRemaining
-        
         clearLaneViews()
         
-        if let allLanes = step.intersections?.first?.approachLanes,
-            let usableLanes = step.intersections?.first?.usableApproachLanes,
-            durationRemaining < RouteControllerMediumAlertInterval {
-            
-            for (i, lane) in allLanes.enumerated() {
-                let laneView = laneArrowView()
-                laneView.lane = lane
-                laneView.maneuverDirection = step.maneuverDirection
-                laneView.isValid = usableLanes.contains(i as Int)
-                stackView.addArrangedSubview(laneView)
-            }
+        let step = currentLegProgress.currentStep
+
+        guard durationRemaining < RouteControllerMediumAlertInterval,
```

Comment:
@bsudekum do you think we should reduce the time threshold for `tertiaryInstructions`?

Reply:
We should not actually be doing any check like this now. The server should tell us exactly when and what to show.